src/main/java/com/atlauncher/utils/ImportPackUtils.java

```diff
@@ -129,7 +129,7 @@ public static boolean loadFromFile(File file) {
                 return loadCurseForgeFormat(file, null, null);
             }
 
-            if (ArchiveUtils.archiveContainsFile(file.toPath(), "index.json")) {
+            if (ArchiveUtils.archiveContainsFile(file.toPath(), "modrinth.index.json")) {
```

Comment:
should this condition run only for `.mrpack` files?

Reply:
Possibly, not a big deal for now, I plan to rework some of the importing things when V2 of Modrinth API comes out anyway as well as part of #425 so not an issue for now